corehq/warehouse/migrations/0033_form_fact_user_id_index.py

```diff
@@ -48,5 +48,10 @@ class Migration(migrations.Migration):
         migrations.RunSQL(
             DROP_INDEX_SQL.format('warehouse_formfact_form_id_1bb74f90_like'),
             migrations.RunSQL.noop
+        ),
+        # "warehouse_formstagingtable_timezone_idx" btree (timezone('UTC'::text, GREATEST(received_on, deleted_on, edited_on)))
+        migrations.RunSQL(
+            DROP_INDEX_SQL.format('warehouse_formstagingtable_timezone_idx'),
```

Comment:
Not clear where this index came from or why it's here. I don't see
any queries in the warehouse or in any reports that would use this
index either on the staging table or the fact table